src/content/docs/licenses/license-information/usage-plans/new-relic-usage-plan.mdx

```diff
@@ -150,222 +150,217 @@ Undefined terms used in this Usage Plan or Order shall have the meaning set fort
     title="List prices"
   >
 
-The following List Price tables identify per-unit prices. For more information, contact your New Relic account representative. 
+The following List Price tables identify per-unit prices. Your Order and/or invoice will include both the Product name and applicable edition for your Buying Program.
```

Comment:
It's just one table. Too nitpicky?

Reply:
i wondered about that too. they had put in that change. thanks